fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/common/BulkDataUtils.java

```diff
@@ -107,7 +110,7 @@ public static String startPartUpload(AmazonS3 cosClient, String bucketName, Stri
     }
 
     public static AmazonS3 getCosClient(String cosCredentialIbm, String cosApiKeyProperty, String cosSrvinstId,
-            String cosEndpointUrl, String cosLocation) {
+            String cosEndpointUrl, String cosLocation, boolean isUseFhirServerTrustStore) {
```

Comment:
isUse is hard to read
I suggest just using `use`
```suggestion
            String cosEndpointUrl, String cosLocation, boolean useFhirServerTrustStore) {
```